turbo.json

```diff
@@ -327,6 +327,7 @@
     "TELEMETRY_DEBUG",
     "TWILIO_MESSAGING_SID",
     "TWILIO_PHONE_NUMBER",
+    "TWILIO_WHATSAPP_PHONE_NUMBER",
```

Comment:
this was missing, couldn't commit without that. Not sure why this was no issue before